serverless.yml

```diff
@@ -21,20 +21,28 @@ provider:
 custom:
   startObservationSchedules:
     TEST: cron(0 12 ? * MON-FRI *)
-    QA: cron(0 14 ? * MON *)
+    QA: cron(0 12 ? * MON *)
+    PROD: cron(0 12 ? * MON *)
   stopObservationSchedules:
     TEST: cron(0 23 ? * MON-FRI *)
     QA: cron(0 23 ? * FRI *)
+    PROD: cron(0 23 ? * FRI *)
   startCaptureSchedules:
     TEST: cron(0 12 ? * MON-FRI *)
-    QA: cron(0 14 ? * MON *)
+    QA: cron(0 12 ? * MON *)
+    PROD: cron(0 12 ? * MON *)
```

Comment:
Is it possible to add comments for these cron schedules just for quick reference?  Also there's this: https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html#CronExpressions